test/call-test.js

```diff
@@ -128,73 +128,59 @@ describe("Call API", function () {
 			nock.enableNetConnect();
 		});
 
-		it("should create a call, promise style", function (done) {
-			client.Call.create(newTestCall)
+		it("should create a call, promise style", function () {
+			return client.Call.create(newTestCall)
 			.then(function (call) {
-				call.to.should.equal(newTestCall.to);
-				call.from.should.equal(newTestCall.from);
-				call.id.should.equal("fakeCallId");
-			})
-			.done(done);
+				call.should.eql(newTestCall);
+			});
 		});
 
-		it("should answer a call", function (done) {
-			client.Call.answer(testCall.id).done(done);
+		it("should answer a call", function () {
+			return client.Call.answer(testCall.id);
```

Comment:
I get what is happening here... but I'd really like some sort of assertion to make it super clear what's going on.  Same for the rest of the `returns`